Packs/Zscaler/Integrations/Zscaler/Zscaler.yml

```diff
@@ -400,6 +401,19 @@ script:
       - 'false'
       required: false
       secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: 'false'
+      description: Whether to retrieve only a list of URL category IDs and names.
+        URLs will not be returned at all (even if displayURL set to true). Note -
+        the API does not support the combination of custom_only and get_ids_and_names_only.
+      isArray: false
+      name: get_ids_and_names_only
```

Comment:
see @bakatzir comment.

Reply:
@bakatzir missed that I have added only this arg (and not the others). Is this anything else to change?